src/services/BadgesService.ts

```diff
@@ -290,4 +291,8 @@ export class BadgesService {
       await reinstateBadge(badgeId)
     })
   }
+
+  static async uploadBadge(title: string, description: string, imgUrl: string, expiresAt?: string) {
```

Comment:
`uploadSpec`?

Reply:
Decided for `uploadSpec` and `createBadgeSpec`